src/main/routes/course.js

```diff
@@ -107,9 +110,18 @@ const course_new_page = async (res, department = false) => {
 /* GET course home page */
 router.route('/')
 	.get(html.auth_wrapper(async (req, res, next) => {
+		const artifacts_active 			=  
+			{ artifact: await Artifact.query().eager('[evaluations, owner.owner.[semester,owner.owner]]'),
+			  formatDate:function(){
+			 	return this.toLocaleString('default', {month: 'short', day: 'numeric', year : 'numeric' } ); 
```

Comment:
Probably want the end to look like `year: 'numeric'});` to match the beginning (there's no space after `(` or `(` and usually no space in `key: value` between `key` and `:`).